auditbeat/auditbeat.reference.yml

```diff
@@ -790,5 +790,9 @@ logging.files:
   # Number of rotated log files to keep. Oldest files will be deleted first.
   #keepfiles: 7
 
+  # The permissions mask to apply when rotating log files. The default value is 0600.
```

Comment:
This one is different than the others. Perhaps you didn't `git add` the final version of this file before committing?

Reply:
@andrewkroh Good catch! Weirdness...I committed by using `git commit -a -m ...` so I don't know how it's possible that the files weren't updated/committed correctly. Ah well...it's fixed now.